roles/result/templates/result.json

```diff
@@ -0,0 +1,7 @@
+{
+  "uuid": "{{ item.item | regex_replace('-.*','') | to_uuid }}",
+  "user": "{{user}}",
+  "spec": {{  _benchmark_example_com_benchmark.spec | to_nice_json  }},
```

Comment:
Shouldn't this just be a list of values?

Reply:
Additionally, `_benchmark_example_com_benchmark` isn't many any assumptions? right? IE the user can pass a different `metadata.name` ?

Reply:
Currently it's not a list of values, but once we finalize on #7 this would change accordingly, and nope only assumption is that the cr's apiVersion is `benchmark.example.com` and the kind is `benchmark`. So if we ever updated our CRD and thus CRs, we'll also have to update this. And yes, it shouldn't matter what `metadata.name` is

Reply:
So it is predicated on some definition we have... The user _could_ change that... i would also recommend we remove "example" from the name.

Reply:
The user can change the cr you mean? Then the operator will not be able to read it or work against it, I think its outside the scope of this PR and even ripsaw. we can't remove example from the name as its how we defined in crd.

Reply:
When the user wants to create the resource based on the cr which if user changes apiversion to say something like `benchmark.exampl2e.com/v1alpha1` then they'd possibly hit an error like `[root@smicro-6029p-04 ripsaw]# kubectl apply -f valid_uperf_store.yaml
error: unable to recognize "valid_uperf_store.yaml": no matches for kind "Benchmark" in version "benchmark.exampl2e.com/v1alpha1"
`